framework/source/class/qx/bom/request/Xhr.js

```diff
@@ -1062,7 +1062,7 @@ qx.Bootstrap.define("qx.bom.request.Xhr",
       if (this._getProtocol() === "file:") {
         error = !this.responseText;
       } else {
-        error = !this.statusText && this.status !== 204;
+        error = this.status === 0;
```

Comment:
@level420 the reason why i committed this was that we really had the problem that our application stopped working with Tomcat 8.5 because of the missing (but RFC Conform Missing) statusText